lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -149,47 +147,57 @@ def fake_metadata_controller(tar_path):
                 "run": {"controller": ""},
             }
 
-        source_tarball, source_md5, md5 = tarball
-        cm = CacheManager(server_config, make_logger)
         # fetching metadata from metadata.log file and key/value not
         # being there should result in a MetadataError
-        tar_name = source_tarball.name.removesuffix(".tar.xz")
-
-        expected_metaerror = f"A problem occurred processing metadata.log from {source_tarball!s}: \"'filename {tar_name}/metadata.log not found'\""
-        monkeypatch.setattr(tarfile.TarFile, "extractfile", fake_extract_file)
-        with pytest.raises(MetadataError) as exc:
-            cm.create(source_tarball)
-        assert str(exc.value) == expected_metaerror
+        source_tarball, source_md5, md5 = tarball
+        cm = CacheManager(server_config, make_logger)
 
         expected_metaerror = f"A problem occurred processing metadata.log from {source_tarball!s}: 'Invalid Tarfile'"
-        monkeypatch.setattr(tarfile, "open", fake_tarfile_open)
-        with pytest.raises(MetadataError) as exc:
-            cm.create(source_tarball)
-        assert str(exc.value) == expected_metaerror
+        # expected_metaerror = f"A problem occurred processing metadata.log from {source_tarball!s}: 'Invalid Tarfile'"
```

Comment:
l.1566 is a commented out l.155 - is it needed?

Reply:
Sigh; missed that. The unit test behavior was really weird for a while and seemed to be changing error messages. The weird thing here is that I ended up with exactly what I'd originally commented out and never noticed. I should fix it, but I guess I'll wait to see if there's any other feedback.

Reply:
Yeah, let's remove this before merging.